modules/ROOT/pages/os-extensions.adoc

```diff
@@ -38,18 +39,27 @@ systemd:
         [Service]
         Type=oneshot
         RemainAfterExit=yes
-        ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive nano
+        # `--allow-inactive` ensure that rpm-ostree does not return an error if
+        # the package is already installed. This is useful if the package is
+        # added to the root image in a future Fedora CoreOS release as it will
+        # prevent the service from failing.
+        ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive vim
         ExecStart=/bin/touch /var/lib/%N.stamp
 
         [Install]
         WantedBy=multi-user.target
 storage:
   files:
-    # use nano as default editor
-    - path: /etc/profile.d/nano.sh
+    # Remove the existing default editor configuration
```

Comment:
Maybe let's not blank out this file and instead write our new preference to something like `/etc/profile.d/z-default-editor.sh` to override it? see https://github.com/coreos/fedora-coreos-config/pull/1394#issuecomment-1007488297